### PR TITLE
Initial import

### PR DESCRIPTION
**Pull request by rdunklau**
Initial import of Connemara

### CODE REVIEW

**Review by marco44 — APPROVED** [review]

File: debian/rules [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+override_dh_auto_build:
+	@echo "Building connemara_replication"
+	cd build && make
+	dh_auto_build $@
+
+override_dh_auto_install:
+	mkdir -p debian/connemara/usr/bin/
+	cp build/connemara_replication debian/connemara/usr/bin/
+	cp replayer/connemara_replay.pl debian/connemara/usr/bin/
+	cp replayer/connemara_rewrite.pm debian/connemara/usr/bin/
```
Comment (PierreDucroquet) [inline debian/rules]: Use `install` instead of `cp` ?

File: connemara_replication/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+SET(MAJOR_VERSION 1)
+SET(MINOR_VERSION 2)
+SET(PATCH_VERSION 0)
+
+
+
+add_definitions("-DCONNEMARA_VERSION=\"${MAJOR_VERSION}.${MINOR_VERSION}.${PATCH_VERSION}\"")
+
+IF(EXISTS "${CMAKE_ROOT}/Modules/CPack.cmake")
```
Comment (PierreDucroquet) [inline connemara_replication/CMakeLists.txt]: Is that still needed?

**Review by PierreDucroquet — DISMISSED** [review]

**Review by marco44 — APPROVED** [review]

File: connemara_replication/CMakeLists.txt (hunk 2 above)
Comment (rdunklau) [reply connemara_replication/CMakeLists.txt]: I removed everything, thanks

File: debian/rules (hunk 1 above)
Comment (rdunklau) [reply debian/rules]: Isn't that the job of dh_auto_install just below ? I'm not really good at packaging ...

Comment (PierreDucroquet) [reply debian/rules]: Nop. install is cp + chmod, and you should use it so your target has the settings you intend, not the ones you get from a local build (and that may be different)

Comment (rdunklau) [reply debian/rules]: Fixed !

**Review by rodo — APPROVED** [review]
Ok for me ! What a PR !
Thank you so much @rdunklau for this non fun part of your job ;-)

**Review by PierreDucroquet — APPROVED** [review]

**Pull request merged**